lib/all_sources.rb

```diff
@@ -0,0 +1,16 @@
+# rubocop:disable Style/ClassVars
+
+# All Sources harvesting utilities
+module AllSources
+
+  # @return [AllSources::Harvester]
+  def self.harvester
+    @@harvester ||= AllSources::Harvester.new
+  end
+
+  def self.logger
+    @@logger ||= Logger.new(Settings.HARVESTER.LOG)
```

Comment:
Does this logger need to rotate?  Is that something we should do here or something we need to have ops do?

Reply:
ops handles all log rotations, but i believe i need to let them know about the new file that is being generated (though this one will be quite small)...I'll do that

Reply:
log rotation confirmed with ops - it will happen based on the .log extension of the file and its location (alongside other log files)